sale_order_team_from_product/readme/CREDITS.md

```
@@ -0,0 +1,2 @@
+The development of this module has been financially supported by:
+  - Geschäftsstelle Sozialinfo
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)